apps/web/playwright/fixtures/types.ts

```diff
@@ -1,4 +1,27 @@
 export enum TimeZoneEnum {
-  USA = "America/Phoenix",
-  UK = "Europe/London",
+  USA = "America/Phoenix", // UTC-7
+  UK = "Europe/London", // UTC
+  ID = "Asia/Jakarta", // UTC+7
+  SAMOA = "Etc/GMT+12", // UTC-12
+  AMERICAN_SAMOA = "Pacific/Pago_Pago", // UTC-11
+  HAWAII = "Pacific/Honolulu", // UTC-10
+  ALASKA = "America/Anchorage", // UTC-9
+  PACIFIC_USA = "America/Los_Angeles", // UTC-8
+  CENTRAL_USA = "America/Chicago", // UTC-6
+  EASTERN_USA = "America/New_York", // UTC-5
+  VENEZUELA = "America/Caracas", // UTC-4
+  ARGENTINA = "America/Buenos_Aires", // UTC-3
+  BRAZIL = "America/Noronha", // UTC-2
+  AZORES = "Atlantic/Azores", // UTC-1
+  FRANCE = "Europe/Paris", // UTC+1
+  GREECE = "Europe/Athens", // UTC+2
+  RUSSIA = "Europe/Moscow", // UTC+3
+  UAE = "Asia/Dubai", // UTC+4
+  PAKISTAN = "Asia/Karachi", // UTC+5
+  BANGLADESH = "Asia/Dhaka", // UTC+6
+  SINGAPORE = "Asia/Singapore", // UTC+8
+  JAPAN = "Asia/Tokyo", // UTC+9
+  AUSTRALIA = "Australia/Sydney", // UTC+10
+  SOLOMON_ISLANDS = "Pacific/Guadalcanal", // UTC+11
+  NEW_ZEALAND = "Pacific/Auckland", // UTC+12
 }
```

Comment:
@anikdhabal I'm just following the existing naming format for the enums, but I recommend using timezone explicit as the key e.g. `UTC`, `UTC_PLUS_7`, etc. for better readability. Or maybe for this test, we can create new enums (?)